CHANGELOG.md

```diff
@@ -1,3 +1,142 @@
+# 1.6.10-beta01 (April 2024)
+
+_Changes since 1.6.2_
+
+## Highlights
+
+- Experimental multiplatform support of `Lifecycle` and `ViewModel`. See [documentation](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-lifecycle.html) for details
```

Comment:
Let's not merge until we publish this doc page

Reply:
Done